cdist/conf/type/__user_groups/gencode-remote

```diff
@@ -23,16 +23,37 @@ state_should="$(cat "$__object/parameter/state")"
 oldusermod="$(cat "$__object/explorer/oldusermod")"
 os=$(cat "$__global/explorer/os")
 
+if [ "$os" = "freebsd" ]; then
+	cmd="pw groupmod \"\$group\" \$action \"\$user\""
+	addparam="-m"
+	delparam="-d"
+elif [ "$os" = "netbsd" ]; then
+	# NetBSD does not have a command to remove a user from a group
+	cmd="usermod \$action \"\$group\" \"\$user\""
+	addparam="-G"
+	delparam=";;#"
```

Comment:
That's what it looked like in the last stable version :D I will re-do the fix against master (which apparently changed -- for better!).